index.js

```diff
@@ -23,6 +24,7 @@ module.exports = {
             create: 'Créer %{name}',
             delete: 'Supprimer %{name} #%{id}',
             dashboard: 'Tableau de bord',
+            not_found: 'Ooops, page manquante.'
```

Comment:
Can you remove the `Ooops` ? I suggest: `Page manquante`

Reply:
Sure :-)